lddtree.sh

```diff
@@ -69,9 +68,11 @@ elf_specs_scanelf() {
 		sed -r 's: (LINUX|GNU)$: NONE:'
 }
 
+
+
 # functions for binutils backend
 elf_rpath_binutils() {
-	objdump -x "$@" | awk '$1 == "RPATH" || $1 == "RUNPATH" { print $2 }'
+	readelf -d "$1" | awk '$2 == "(RPATH)" || $2 == "(RUNPATH)" { print $5 }' | cut -d '[' -f 2 | sed 's/]//'
```

Comment:
- readelf can retrieve any needed information, objdump only a few. Some custom linux distro do not rely on package system or have "binutils" package. To limit dependency on several tool, use only one tool

- Right, should be better to use awk only. I am not so good using awk. I will try